Services/Simulation/InternalInterpreter.cs

```diff
@@ -47,43 +48,42 @@ public InternalInterpreter(ILogger logger)
             this.random = new Random();
         }
 
-        public Dictionary<string, object> Invoke(
+        public void Invoke(
```

Comment:
So looks like we don't need return values?

Reply:
Yes. The Invoke method already modifies the value passed in and so doesn't also need to return the value and assign it.